code-challenges.test.js

```diff
@@ -39,12 +90,36 @@ var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 
 
 // b) Create the function that makes the test pass.
-
-
-
+// Declare a function called odds that takes in arrays as a parameter
+// Declare a variable oddOnly equal to an empty array for storing our odd numbers later
+// For loop for iteration
+// Conditional if array[i] % 2 does not equal 0
+// push array[i] into oddOnly
+// Return oddOnly sorted from least to greatest AND filtering typeof by 'number'
+
+const odds = (array) => {
+    let oddOnly = []
+    for(let i=0; i<array.length; i++){
+        if(array[i] % 2 !== 0){
```

Comment:
This is all a good set up for this line of code. Whats great about higherorder functions is that you can take this line evaulation and append it onto the line 106  "typeof value === "number"" with an && and YATZEE you are done!

Reply:
```javascript
const odds = (array) => {
return array.sort((a, b) => a - b).filter(value => typeof value === "number" && value % 2 !== 0)
}
```